pom.xml

```diff
@@ -88,6 +101,7 @@
 			</plugin>
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
+				<!-- TODO maven says: not found-->
```

Comment:
@Dean442  ist das bei dir in Intellij auch rot markiert?

Reply:
nope bei mir ist alles brav da

Reply:
ok seltsam...dann lass ichs drin